webrtc-stats.html

```diff
@@ -2711,6 +2713,26 @@ <h2>
                   but is a single concealment event.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>deceleratedSamples</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  The total number of samples inserted in order to decelerate playout speed. This
+                  can happen to prevent the jitter buffer from becoming empty.
```

Comment:
Alternate text: "When playout is slowed down, this counter is increased by the difference between the number of samples received and the number of samples played out in the interval. If playout is slowed down by inserting samples, this will be the number of inserted samples."

Reply:
Name: insertedSamples
Rephrase: Decelerating is a consequence, not a definition.

Reply:
I talked with Chen and things to note:

- Samples added or removed for acceleration/deceleration may be based on true samples. When "removing samples" we are not simply discarding samples, how we end up with less samples is up to the implementation and may involve analysing all of the samples that we did receive.
- If the name is as vague as "insert/add" it sounds like this would include concealedSamples, which is not the case.
- If the name is as vague as "remove/discard" this sounds like it would include buffer flushes, which is not the case.
- We are talking about samples for playout, not adding or removing to the jitter buffer per-se.

Conclusion:
- Harald's definition is good.
- The name should include "accelerate/decelerate" or similar, to avoid confusion with other reasons for adding or removing.

Reply:
Suggestion:

Name: insertedAcceleratedSamples
Description: Harald's definition